attest/verifier/src/lib.rs

```diff
@@ -145,9 +146,13 @@ pub enum Error {
     BadSignature,
     /// There was an error parsing the JSON contents: {0}
     Parse(VerifyError),
+
+    /// Attestation verifier error: {0}
+    AttestationVerify(AttestationVerifierError),
```

Comment:
Up for suggestions on this, this error variant is more or less the DCAP versions of the other variants in this error.